vavr/src/test/java/io/vavr/collection/AbstractSeqTest.java

```diff
@@ -1328,22 +1328,20 @@ public void shouldReturnSameSeqWhenEmptyRemoveAllNonEmpty() {
 
     // -- removeAll(Predicate)
 
-    @SuppressWarnings("deprecation")
     @Test
     public void shouldRemoveExistingElements() {
         final Seq<Integer> seq = of(1, 2, 3);
-        assertThat(seq.removeAll(i -> i == 1)).isEqualTo(of(2, 3));
-        assertThat(seq.removeAll(i -> i == 2)).isEqualTo(of(1, 3));
-        assertThat(seq.removeAll(i -> i == 3)).isEqualTo(of(1, 2));
-        assertThat(seq.removeAll(ignore -> true)).isEmpty();
-        assertThat(seq.removeAll(ignore -> false)).isSameAs(seq);
+        assertThat(seq.reject(i -> i == 1)).isEqualTo(of(2, 3));
```

Comment:
This test must use exactly `removeAll` method. It is deprecated, but must be tested too